src/modules/attributes.ts

```diff
@@ -26,28 +26,31 @@ function updateAttrs(oldVnode: VNode, vnode: VNode): void {
   if (oldAttrs === attrs) return;
   oldAttrs = oldAttrs || {};
   attrs = attrs || {};
+  const lowAttrs: any = {};
 
   // update modified attributes, add new attributes
   for (key in attrs) {
     cur = attrs[key];
-    old = oldAttrs[key];
+    const lowkey = key.toLowerCase();
```

Comment:
if this is because `toVNode()` and not really about folks using snabbdom incorrectly, then the lowercasing should happen during `toVNode()` routine instead instead of doing it during the critical path.